pgx-version-updater/Cargo.toml

```diff
@@ -0,0 +1,19 @@
+[package]
+name = "pgx-version-updater"
+version = "0.0.0"
+edition = "2021"
+authors = ["ZomboDB, LLC <zombodb@gmail.com>"]
+license = "MIT"
+description = "Standalone tool to update PGX Cargo.toml versions and dependencies in preparation for a release."
+homepage = "https://github.com/zombodb/pgx"
+repository = "https://github.com/zombodb/pgx"
+documentation = "https://docs.rs/pgx-utils"
+readme = "README.md"
+rust-version = "1.58"
+
+[dependencies]
+clap = { version = "3.2.2", features = [ "env", "derive" ] }
+owo-colors = "3.4.0"
+regex = "1"
```

Comment:
You don't seem to actually use this in the crate.
```suggestion
```

Reply:
🤦‍♂️